### ISSUE TITLE
Modal is closing when press space

### ISSUE THREAD

**Issue by ridvandmrc** [siemens/ix#1077]
### Prerequisites

- [X] I have not leaked any internal company information.
- [X] I have not posted internal links or code snippets.

### What happened?

I have switch button in modal, and **closeOnBackdropClick:true**,
If I focus the switch button and press **space to change status**. modal is closing **immediately**

### What type of frontend frameware are you seeing the problem on?

Angular

### Which version of iX do you use?

latest

### Code to produce this issue.

```shell
you can check here
https://stackblitz.com/~/edit/xnfrdi-hwdxjf
```

**Comment by matthiashader** [comment]
Internal ref [IX-908]

**Comment by danielleroux** [comment]
A fix for this issue is now available in v2.1.1-beta.0.
We kindly ask you to install this version in your dev environment to check if it resolves the problem for you and give us some feedback about this.

**Comment by ridvandmrc** [comment]
Hi @danielleroux ,
It is working like a charm. 
many thanks
